File: BotsSDK/korebot/src/main/java/kore/botssdk/utils/LogUtils.java
package kore.botssdk.utils;

public class LogUtils
{
    static boolean LOG = false;

    public static void enableLog(boolean enable) {
        LOG = enable;
    }

    public static void info(String tag, String string) {
        if (LOG) android.util.Log.i(tag, string);
    }
    public static void error(String tag, String string) {
        if (LOG) android.util.Log.e(tag, string);
    }
    public static void debug(String tag, String string) {
        if (LOG) android.util.Log.d(tag, string);
    }
    public static void verbose(String tag, String string) {
        if (LOG) android.util.Log.v(tag, string);
    }
    public static void warn(String tag, String string) {
        if (LOG) android.util.Log.w(tag, string);
    }

    public static boolean isEnabled()
    {
        return LOG;
    }
}

File: BotSdk/botclient/src/main/java/com/kore/common/utils/LogUtils.kt
package com.kore.common.utils

import android.util.Log
import com.kore.botclient.BuildConfig

object LogUtils {
    private var isEnabled = BuildConfig.DEBUG
    fun enableLog(enable: Boolean) {
        isEnabled = enable
    }

    fun i(tag: String, string: String?) {
        if (isEnabled) Log.i(tag, "$string")
    }

    fun e(tag: String, string: String?) {
        if (isEnabled) Log.e(tag, "$string")
    }

    fun d(tag: String, string: String?) {
        if (isEnabled) Log.d(tag, "$string")
    }

    fun v(tag: String, string: String?) {
        if (isEnabled) Log.v(tag, "$string")
    }

    fun w(tag: String, string: String?) {
        if (isEnabled) Log.w(tag, "$string")
    }
}